samples/ncp/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
For coherence with the other samples, we can update the copyright dates to 2024 in NCP sample files.